dd-sdk-android/src/main/kotlin/com/datadog/android/_InternalProxy.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android
+
+import com.datadog.android.telemetry.internal.Telemetry
+
+/**
+ * This class exposes internal methods that are used by other Datadog modules and cross platform
+ * frameworks. It is not meant for public use.
+ *
+ * DO NOT USE this class or its methods if you are not working on the internals of the Datadog SDK
+ * or one of the cross platform frameworks.
+ *
+ * Methods, members, and functionality of this class  are subject to change without notice, as they
+ * are not considered part of the public interface of the Datadog SDK.
+ */
+@Suppress(
+    "UndocumentedPublicClass",
+    "UndocumentedPublicFunction",
+    "UndocumentedPublicProperty",
+    "ClassNaming",
+    "VariableNaming"
+)
+class _InternalProxy {
```

Comment:
Yeah, it's a good point. My thinking was that we might want Telemetry without RUM -- or I guess more importantly we might initialize and send telemetry even if the end user doesn't initialize RUM, so I wanted to keep it in the Core internals rather than on the RUM one.